continuous-integration/reference-architectures/README.md

```diff
@@ -1,3 +1,26 @@
 # Reference Architectures
 
-Nothing here yet, come back soon... 
\ No newline at end of file
+## Overview
+These reference architectures are meant as a starting point for implementing SLIM practices. 
+Each one correlates to a specific use case described in the SLIM use case diagram. Directories 
+presented here are intended to reflect the organization nature of suggested use cases and are 
+named correspondingly. Organization is segmented by use case section and then segmented by 
+the development language used to implement the solution. Please feel free to implement, modify, 
+and extend the samples presented to fit particular needs. 
+
+## Examples
+**compilation-release/java-maven** | A basic Java application build is presented here that uses 
```

Comment:
Hey @jpl-jengelke - I think examples like these are best suited to be placed in the starter kits section. Take a look at the [Adding a New Best Practice Guide guidance page](https://nasa-ammos.github.io/slim/contributing/#adding-a-new-best-practice-guide), specifically by scrolling down to the `Reference Architectures` section. You'll notice the Reference Architectures are supposed to include only diagrams, whereas the `Starter Kits` section explains templates / code is supposed to live there. 

Can you scope out those the guide linked above and apply that to your work here? I think that would help people understand context better and use your templates more effectively.

Reply:
It has been moved. Sorry, I did not see the comment in the conversation.